modules/apps/foundation/users-admin/users-admin-web/src/main/resources/META-INF/resources/view_flat_users.jsp

```diff
@@ -266,7 +266,13 @@ boolean showRestoreButton = (searchTerms.getStatus() != WorkflowConstants.STATUS
 		}
 		else {
 			if (filterManageableOrganizations && !UserPermissionUtil.contains(permissionChecker, ResourceConstants.PRIMKEY_DNE, ActionKeys.VIEW)) {
-				userParams.put("usersOrgs", ArrayUtil.toLongArray(user.getOrganizationIds()));
+				long[] organizationIds = user.getOrganizationIds();
+
+				if (ArrayUtil.isEmpty(organizationIds)) {
+					organizationIds = new long[] {0};
```

Comment:
Hey Jorge,

In line 272, it's not an empty array. It's an array with the default organization id 0: `{0}`.